include/powsybl/iidm/ThreeWindingsTransformer.hpp

```diff
@@ -183,14 +70,14 @@ class ThreeWindingsTransformer : public Connectable {
     const std::string& getTypeDescription() const override;
 
 private:
-    friend class ThreeWindingsTransformerAdder;
+    friend class three_windings_transformer::Leg;
```

Comment:
Why the Leg is friend?

Reply:
It's in order for `Leg` to have access to `getTerminal` which is inherited protectedly from `Connectable`.